### PR TITLE
arash/DTrade static

### PR DESCRIPTION
**Pull request by arash-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update

### CODE REVIEW

File: src/pages/dtrader/_experience.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        margin-top: 3.56rem;
+    }
+`
+const Experience = () => {
+    return (
+        <StyledSection padding="4rem 0">
+            <StyledContainer justify="space-between">
+                <Content>
+                    <StyledHeader as="h2" align="center">
+                        20 years of experience and trust
```
Comment (oskar-binary) [inline src/pages/dtrader/_experience.js]: Localize is missing for a lot of text

File: src/pages/dtrader/_experience.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                    </StyledHeader>
+                    <Show.Mobile>
+                        <Text align="center">
+                            Start trading on DTrader now.
+                        </Text>
+                    </Show.Mobile>
+                    <Show.Desktop>
+                        <Text align="center">Check out our live demo</Text>
+                    </Show.Desktop>
+                    <StyledButton demo>Go to live demo</StyledButton>
```
Comment (oskar-binary) [inline src/pages/dtrader/_experience.js]: Same here, this button should link to the live demo

File: src/pages/dtrader/_experience.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        <StyledSection padding="4rem 0">
+            <StyledContainer justify="space-between">
+                <Content>
+                    <StyledHeader as="h2" align="center">
+                        20 years of experience and trust
+                    </StyledHeader>
+                    <Text align="center">
+                        Join over a million traders worldwide
+                    </Text>
+                    <StyledButton secondary>Try for free</StyledButton>
```
[the remark this answers is not in the document]
Comment (arash-fs) [reply src/pages/dtrader/_experience.js]: Done

File: src/pages/dtrader/_experience.js (hunk 1 above)
Comment (arash-fs) [reply src/pages/dtrader/_experience.js]: Done

File: src/pages/dtrader/_experience.js (hunk 2 above)
Comment (arash-fs) [reply src/pages/dtrader/_experience.js]: Done

**Comment by ashkanx** [comment]
@arash-fs conflicts

**Pull request merged**